Build 1

Anigilator322 opened the pull request.

Anigilator322 committed:
refs

Anigilator322 committed:
ъ

Anigilator322 committed:
reset

Anigilator322 committed:
HELLL YEAH

Anigilator322 committed:
Merge branch 'main' into FoWRefactorForAI

Anigilator322 committed:
zabbix fix

Anigilator322 committed:
Update Prefabs_1.asset

Anigilator322 committed:
Merge branch 'MemoryLeak' into Build_1

Anigilator322 committed:
build reset fix

Anigilator322 committed:
a

Anigilator322 committed:
refactor fow

Anigilator322 committed:
я

Anigilator322 committed:
zabix

CapSparkle commented on Assets/Scripts/Bootstrap/Installers/GameInstaller.cs:
```diff
@@ -28,10 +28,13 @@ public class GameInstaller : MonoInstaller
         {
             // ======= Models =======
             Container.Bind<Soil>().FromComponentInNewPrefab(generalPrefabs.soilPrefab).AsSingle();
+            Container.Bind<Renderer>().FromComponentInNewPrefab(generalPrefabs.FogOfWarPrefab).AsSingle()
```
Такие фундаментальные вещи надо биндить с Id

CapSparkle commented on Assets/Scripts/FogOfWar/VisibilityModel.cs:
```diff
@@ -0,0 +1,32 @@
+        #region VisibilitySystem
+        public List<VisibilityCapsule> VisibilityCapsules;
+        public Dictionary<Plant, List<IPositionedObject>> VisibleByPlantsPoints;
+        public PlantsModel PlantsModel;
```
Вынести связи с PlantsModel и Soil.

CapSparkle commented on Assets/Scripts/FogOfWar/VisibilitySystem.cs:
```diff
@@ -1,44 +1,31 @@
     public class VisibilitySystem
     {
-        private PlantsModel _plantsModel;
+        public VisibilityModel VisibilityComponent { get; set; }
```
Переименовать в visibilityModel

CapSparkle commented on Assets/Scripts/FogOfWar/VisibilitySystem.cs:
Сделать так чтобы видимость появлялась даже если у ноды нет Parent-ноды

Anigilator322 committed:
FOW init

Anigilator322 committed:
resources

Anigilator322 committed:
Update PlayerRootBuildingInput.cs

Anigilator322 committed:
cameraMove

Anigilator322 committed:
Update CameraMoveInput.cs

CapSparkle committed:
Pre-release Polishing 1

- soil borders for root building
- gray background over soil bounds
- root building block while clicking root type selections

Anigilator322 committed:
Merge pull request #47 from Anigilator322/Build_1_adjustments_1

Pre-release Polishing 1

CapSparkle committed:
Merge pull request #46 from Anigilator322/BugFixes

Bug fixes

Anigilator322 committed:
asdasdas

Anigilator322 committed:
Update SceneryAgent.cs

CapSparkle committed:
two plants with different positions created

CapSparkle committed:
Merge pull request #48 from Anigilator322/Build_1_two_plants

two plants with different positions created

Anigilator322 committed:
fas

Anigilator322 committed:
Update VisibilitySystem.cs

CapSparkle committed:
Merge pull request #49 from Anigilator322/FOW_FIX_AGAIN

fas

CapSparkle committed:
Merge branch 'Build_1' into TUPOY_AI

CapSparkle committed:
GAMEPLAY WORKS

CapSparkle committed:
Merge pull request #50 from Anigilator322/TUPOY_AI

Tupoy ai

Anigilator322 committed:
roots fix

Anigilator322 committed:
fix Blueprints
